### PR TITLE
Quick start optimizations/clarifications

### PR DESCRIPTION
**Pull request by eatonphil**
Moves all READMEs to all for Copy-ing the contents when possible by splitting out stdout/stderr of a command into a separate code block.

### CODE REVIEW

File: bootstrap.sh [hunk 1]
```diff
@@ -34,11 +37,18 @@
 	unzip -qo "tigerbeetle-$arch-$os-$version.zip"
 	chmod +x tigerbeetle
     else
+	echo "Building TigerBeetle binary from source for your machine."
```
Comment (matklad) [inline bootstrap.sh:40]: :heart:, that's super important feedback for the user!

File: docs/quick-start/node-cli.md [hunk 2]
```diff
@@ -10,13 +10,13 @@ running server and do some accounting!
 First install the Node client.
 
 ```javascript
```
Comment (matklad) [inline docs/quick-start/node-cli.md]: this is not javascript though

Comment (eatonphil) [reply docs/quick-start/node-cli.md]: Fixed, thank you!

File: src/clients/README.md [hunk 3]
```diff
@@ -113,7 +117,11 @@ directory where the sample code is copied into and where munging takes
 place.
 
 ```bash
+<<<<<<< HEAD
```
Comment (matklad) [inline src/clients/README.md]: merge conflict

File: src/clients/go/samples/basic/README.md [hunk 4]
```diff
@@ -41,8 +40,7 @@ address of the TigerBeetle server you started.
 Now you can run this sample:
 
 ```console
-$ go run main.go
-```
+go run main.go```
```
Comment (matklad) [inline src/clients/go/samples/basic/README.md]: Misplaced closing backtics

**Comment by eatonphil** [comment]
@matklad should be good!

**Review by matklad — APPROVED** [review]
:+1:

**Pull request merged**